narratives/ncov_sit-rep_2020-03-20.md

```diff
@@ -0,0 +1,2453 @@
+First, the map shows very few sequences from the global south. For example, we only have sequences from 4 of the 25 countries in Latin America where the WHO reports cases of COVID-19. This is NOT because COVID-19 isn't circulating in these areas, or that these cases are not as crucial to understand; rather, we just don't have much data available from these areas.
+<br><br>
+Conversely, the map shows many sequences from the Netherlands compared to the rest of Europe. However, looking at the WHO's total case counts, the confirmed cases in the Netherlands actually make up about 2\% of all the confirmed cases in Europe.
+<br><br>
+So, when you're reading this narrative, keep in mind that the size of each circle on the map indicates how much data is currently available from that area, rather than the true size of the outbreak. You can [read here](https://nextstrain.org/narratives/trees-background?n=8) about how this can effect our inferences of transmission.
+
+
+<!-- There is NO right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 5 -->
+# [Sequence data can cross-check travel histories](https://nextstrain.org/ncov/2020-03-19?legend=closed&&c=division_exposure&label=clade:A1a&d=tree,map)
+
+Identifying where a given case was infected is important for understanding which areas are experiencing local transmission versus primarily travel-associated cases. Sequence data can help us differentiate between these scenarios.
+<br><br>
+Here, the tree is colored by the travel history of a case (when known). Mousing over the blue Canadian sequence in the middle, we see that they have a reported travel history to Europe. This infection groups with other European sequences, showing that this case almost certainly is a travel-related infection.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 6 -->
+# [Sequence data can cross-check travel histories](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=division_exposure&d=tree,map&p=full)
+
+Zooming out, we can see that this isn't always so consistent.
+For example, mousing over the bright green Canadian sequence in the top right (near clade A7), we see that this case has a travel history to Iran.
+However, this sequence doesn't group anywhere near the rest of the cases with Iranian travel history! This makes it much less clear where this person was actually infected, despite the travel history.
+
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text 7 -->
+# [The virus has been introduced to nearly every country around the world](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=country&d=tree,map&p=grid)
+
+On the tree, we see lots of this kind of intermingling of samples from different countries.
+This indicates that this kind of inevitable human movement has already introduced the virus to many parts of the globe.
```

Comment:
"On the tree, we see lots of ~this kind of~ intermingling..."
"This indicates ~that this kind of~ inevitable human movement..."